### DISCUSSION TITLE
snap shots not updating in home assistant

### DISCUSSION THREAD

**Discussion by robmiller1000** [mrlt8/docker-wyze-bridge#710] [General]
I have 4 cameras running on wyze docker locally ( v3 pan, v3 pro, outdoor v2 and original pan). My v3 pro seems to never fail loading new snapshots. The other seem to fail all the time. If i click for a stream they will update but they will not on there own. They were all added same way to HA frontend card. Any suggestions?

**Comment by mrlt8** [comment]
Do you have on-demand enabled? The bridge will try to avoid waking the cameras if on-demand is enabled unless it's missing a snapshot.

You can manually pull a new snapshot using the webUI endpoint: `:5000/snapshot/camera-nickname.jpg`

**Reply by robmiller1000** [reply]
Thank you so much. That was the issue
